src/status_im/accounts/handlers.cljs

```diff
@@ -107,7 +107,11 @@
                       (map (fn [{:keys [address] :as account}]
                              [address account]))
                       (into {}))]
-    (assoc db :accounts accounts)))
+    (-> db
+        (assoc :accounts accounts)
+        (assoc :view-id (if (empty? accounts)
+                          :chat
+                          :accounts)))))
```

Comment:
it could be just
```clojure
(assoc db :accounts accounts
          :view-id (if (empty? accounts)
                     :chat
                     :accounts))